CMakeLists.txt

```diff
@@ -112,11 +112,24 @@ endif()
 
 set(LLVM_SPIRV_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/include)
 
+# first try locating SPIRV-Tools via pkgconfig (the old way)
 pkg_search_module(SPIRV_TOOLS SPIRV-Tools)
-if(NOT SPIRV_TOOLS_FOUND)
-  message(STATUS "SPIRV-Tools not found; project will be built without "
+if (NOT SPIRV_TOOLS_FOUND)
+  # then try locating SPIRV-Tools via cmake (the new way)
+  find_package(SPIRV-Tools)
+  find_package(SPIRV-Tools-tools)
+  if (SPIRV-Tools_FOUND AND SPIRV-Tools-tools_FOUND)
```

Comment:
It's a first time I see spelling like SPIRV-Tools-tools and googling doesn't help. When I can learn more about this package? If it's the same as SPIRV-Tools, shouldn't we have `OR` instead of `AND` in `if` statement?

Reply:
Hi, @MrSidims!

This is a separate package for the binary executables of the standard SPIRV-Tools. I have added the patch upstream for that just yesterday, so there is no google results for that: https://github.com/KhronosGroup/SPIRV-Tools/pull/5034

Basically, SPIRV-Tools define a separate package for each ot their subfolders. This one is for the subfolder `./tools/`

Reply:
Basically all my comments can be replaced with: do we need both `find_package(SPIRV-Tools)` and `find_package(SPIRV-Tools-tools)` and if yes, how `..._FOUND` should be used in the checks?

Reply:
@dimula73 got it, thanks! Feel free to ignore the comments below